phpcs.xml

```diff
@@ -6,6 +6,9 @@
 <ruleset name="MediaWiki">
     <rule ref="Generic.Classes" />
     <rule ref="Generic.CodeAnalysis" />
+    <rule ref="Generic.CodeAnalysis.UnusedFunctionParameter.Found">
+        <severity>0</severity>
+    </rule>
```

Comment:
@gbirke I'm disabling this again, as this rule just has too many false positives. PhpStorm has a sniff to find this kind of stuff which is somewhat more intelligent - it will not shout at you in places where an interface or base class forces a param to be present.